app/index.js

```diff
@@ -126,6 +126,10 @@ module.exports = yeoman.Base.extend({
     },
 
     gdtBase: function() {
+      // Save the existing composer.json file if any. See composerJson().
+      if (this.fs.exists('composer.json')) {
+        this.fs.copy('composer.json', 'composer.orig');
```

Comment:
This is still relevant.  Need to refactor to just load values into memory and let generator rewrite as needed.